app/main.py

```diff
@@ -1,6 +1,33 @@
+import os
+
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+URL = f"http://api.weatherapi.com/v1/current.json?key={API_KEY}&q={CITY}"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    if API_KEY is None:
+        raise Exception("API Key not found.")
+    response = requests.get(URL)
+    if response.status_code == 200:
+        retrieve_result(response)
+    else:
+        raise Exception("Bad request. Check API key and WeatherAPI status")
```

Comment:
bad request is status code 400, consider using more abstract name

Reply:
It is console app, not a django app, do you really reccomend to use django's errors here?